docs/native-modules-setup.md

```diff
@@ -45,154 +45,64 @@ The steps to create a new native module library project are:
-| Mso                                 | `Mso\Mso.vcxitems`                                                               |
-| ReactCommon                         | `ReactCommon\ReactCommon.vcxproj`                                                |
-| ReactWindowsCore                    | `ReactWindowsCore\ReactWindowsCore.vcxproj`                                      |
-| Shared                              | `Shared\Shared.vcxitems`                                                         |
-
-For each project, you'll do the following:
+```bat
+npm info react-native-windows@canary devDependencies.react
+```
 
-1. Open the Solution Explorer sidebar.
-1. Right-click on the `ReactNative` folder.
-1. Select `Add` > `Existing Project...`.
-1. Select the project file and click `Open`.
+Take the result of that command (let's say it's `16.13.1`) and use it to upgrade the dev dependency:
 
-When you are done, your solution should look like this:
-![native module dependencies](assets/native-module-dependencies.png)
+```bat
+yarn upgrade react@16.13.1 --dev
+```
 
-You now have all of the React Native Windows projects to your solution. Next we're going to reference them in our `MyLibrary` project.
+You'll need to repeat the steps for `react-native`, ie:
 
-### Referencing React Native Windows in your Project
+```bat
+npm info react-native-windows@canary devDependencies.react-native
+```
 
-The only project reference you **must** add is `Microsoft.ReactNative`. To add the reference:
+Again, take the result of that command (let's say it's `0.0.0-a36d9cd7e`) and use it to upgrade the dev dependency:
 
-1. Open the Solution Explorer sidebar.
-1. Right-click on your `MyLibrary` project.
-1. Select `Add` > `Reference`.
-1. Select `Projects` on the left-hand side.
-1. Check the box next to `Microsoft.ReactNative`.
-1. Click `OK`.
+```bat
+yarn upgrade react-native@0.0.0-a36d9cd7e --dev
+```
 
-After you've added the reference, you need to make sure it doesn't copy itself into your build (otherwise it'll cause build conflicts down the line when you're trying on consume your library):
+Now you should be ready to add Windows support with the CLI.
 
-1. Open the Solution Explorer sidebar.
-1. Under your `MyLibrary` project, expand the `References`.
-1. Right-click on `Microsoft.ReactNative`.
-1. Select `Properties`.
-1. Under `Build`, Change `Copy Local` to `False`.
+## Adding Windows support to an existing library
 
-Now, you're technically ready to go, but in order to improve the developer experience, it's also **highly recommended** to also add a reference to the appropriate helper shared project. These projects contain the attributes (C#) and macros (C++) as described in the [Native Modules](native-modules.md) and [View Managers](view-managers.md) documents.
+Similar to adding Windows support to an app project, you can use the CLI to add Windows support to a native module project, but you'll need to specify `--projectType lib`:
```

Comment:
I'm not sure if there's a good solution, but I remember confusion before when someone tried running commands in our docs at the root of a monorepo. Not sure if adding something about "run this in the directory with your module package" would be useful or not.

Reply:
I can make the language clearer that the location to run the command is usually the repo root but not always.